netmanager/src/utils/dateTime.js

```diff
@@ -13,9 +13,7 @@ export const formatDateString = (
 
 export const getElapsedDurationMapper = (dateTimeStr) => {
   let delta =
-    Math.abs(
-      new Date() - new Date(moment.utc(dateTimeStr).tz(moment.tz.guess()))
-    ) / 1000;
+    Math.abs(moment.utc(new Date()) - moment.utc(new Date(dateTimeStr))) / 1000;
```

Comment:
Thanks, also confirmed (using the endpoint) that the time is actually 2 hours ago. I am now curious if there is something we need to review from the data pipeline perspective. Is it okay for ALMOST all devices to have "last refreshed 2 hours" in actual sense?

Screenshot is of a current device showing last refreshed 2 hours ago and it is accurate! It is now 11:09 am.
<img width="796" alt="Screenshot 2022-03-08 at 11 01 38" src="https://user-images.githubusercontent.com/1590213/157193935-5f3b00bd-1106-48f6-a7d7-15180d5f7112.png">

Reply:
@Baalmart I am thinking this could and one hour and some minutes that is rounded to 2 hours. Could be the reason for the often seen 2 hours.